### DISCUSSION TITLE
Is it possible to restore the GTK notification icon?

### DISCUSSION THREAD

**Discussion by Br31zh** [baresip/baresip#3242] [Q&A] [answered]
Hello,

Since the v3.17.0, the GTK module have replaced the notification zone icon by a standalone window. Is it possible to restore the previous behaviour via configuration? Or has the ability to run baresip in the background been completely removed?

**Comment by robert-scheck** [comment, marked as answer]
As per commit 085e86fa32f67f6fe54b92cb11aa8ee740032a53, there are two configuration options for `~/.baresip/config`:

 - `gtk_use_status_icon` (boolean)
 - `gtk_use_window` (boolean)

If I understand you correctly, the desired configuration is:

```
gtk_use_status_icon   yes
gtk_use_window        no
```

Above example is the classic behaviour before commit 085e86fa32f67f6fe54b92cb11aa8ee740032a53, which works for me during daily usage. However, `gtk_use_status_icon` indeed requires a desktop environment that supports status icons (either natively or using a plugin/addon or similar).

**Comment by Br31zh** [comment]
I missed that, thanks a lot!
